data-products/src/firefox_newtab/sql/glean_firefox_new_tab_impressions_hourly/glean_firefox_new_tab_daily_engagement_by_tile_id_position/data.sql

```diff
@@ -48,6 +48,9 @@ WITH
         'recommendation_id') IS NOT NULL
       OR mozfun.map.get_key(e.extra,
         'tile_id') IS NOT NULL)
+    --filter out records where scheduled corpus item ID is populated to prevent double count
+    AND mozfun.map.get_key(e.extra,
+        'scheduled_corpus_item_id') IS NULL
```

Comment:
what about not null scenario. where is that counted ?

Reply:
yes, that's happening [here](https://github.com/Pocket/data-flows/blob/c88aac8df8b8fe1bda351048671a723d0760efce/data-products/src/firefox_newtab/sql/glean_firefox_new_tab_impressions_hourly/glean_firefox_new_tab_daily_engagement_by_scheduled_corpus_item_id_position_country_locale/data.sql#L47)

Reply:
@chidamnat in addition to what Chelsey mentioned, this change relates to only pre-merino engagement data.

For Firefox 129 and later (Merino based recommendations), the data extracts are happening in another Prefect flow (as Chelsey mentioned above).

Finally in the data models we are merging the engagement data from the new Merino and prior data in the data model `firefox_new_tab_daily_engagement_by_scheduled_corpus_item_id_position_source_country_locale_x`, the one that you have used recently to update a metaflow job